web_accounts_list.json

```diff
@@ -220,6 +253,17 @@
         "category" : "images",
         "valid" : true
       },
+      {
+        "name" : "Cytoid",

```

Comment:
This site does not work for me. Gives me a 404 for this known_user `Cytoid --> Bad detection code and string. Received Code: 404; Expected Code: 200.`

Reply:
This was caused due to a silly typo on my end. That's my bad for not realizing it